idioms/README.md

```diff
@@ -0,0 +1 @@
+# Idioms
```

Comment:
Same: We need to think about a description for this category

Reply:
Just add a `TODO: add description` as a placeholder in all the new readmes, so that it's clear what we want to do.
Then we open an issue "Add chapters description".